Configs/.config/hypr/scripts/screenshot.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 if [ -z "$XDG_PICTURES_DIR" ] ; then
-    XDG_PICTURES_DIR="$HOME/Pictures"
+    XDG_PICTURES_DIR="$HOME/Pictures/screenshots/"
```

Comment:
I think we should use    ```  XDG_PICTURES_DIR="$HOME/Pictures" ``` here

And 

put the folder  in ``` save_dir="${2:-$XDG_PICTURES_DIR/Screenshots}```
And use sentence case for the directory.